target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -2,34 +2,30 @@ do_sysinfo_octeon() {
 	local machine
 	local name
 
-	machine=$(grep "^system type" /proc/cpuinfo | sed "s/system type.*: \(.*\)/\1/g")
-
-	case "$machine" in
-	"UBNT_E100"*)
-		name="erlite"
-		;;
-
-	"UBNT_E200"*)
-		name="er"
-		;;
-
-	"UBNT_E220"*)
-		name="erpro"
-		;;
-
-	"ITUS_SHIELD"*)
-		name="itus,shield-router"
-		;;
-
-	*)
-		name="generic"
-		;;
+	machine="$(awk '/system type/ {print $4}' /proc/cpuinfo)"
+
+	case "${machine}" in
+		"UBNT_E100")
+			name="erlite"
+			;;
+		"UBNT_E200")
+			name="er"
+			;;
+		"UBNT_E220")
+			name="erpro"
+			;;
+		"ITUS_SHIELD")
+			name="itus,shield-router"
+			;;
+		*)
+			name="generic"
+			;;
 	esac
 
-	[ -e "/tmp/sysinfo/" ] || mkdir -p "/tmp/sysinfo/"
+	mkdir -p "/tmp/sysinfo"
 
-	echo "$name" > /tmp/sysinfo/board_name
-	echo "$machine" > /tmp/sysinfo/model
+	echo "${name}" > /tmp/sysinfo/board_name
```

Comment:
Why?

Reply:
becuase of curly brackets.
easier to understand that this thing inside them should expand into the variable.
and less chance to screw it if someone wants to expand that call.

it is just a good practice to use them at all times then don't.

Reply:
Like above, brackets or no brackets is a matter of taste here, so I prefer not changing it without a reason.